### PR TITLE
Indicate HSI attributes that will only be returned for specific CPU v…

### PR DESCRIPTION
**Pull request by hughsie**
…endors

Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [ ] Code fix
- [ ] Feature
- [X] Documentation

### CODE REVIEW

**Review by superm1 — DISMISSED** [review]

File: docs/hsi-tests.d/org.fwupd.hsi.Amd.SpiReplayProtection.json [hunk 1]
```diff
@@ -14,5 +14,8 @@
     "enabled": "SPI protections enabled"
   },
   "hsi-level": 2,
+  "capabilities": [
+    "cpu-amd"
+  ],
```
Comment (hughsie) [inline docs/hsi-tests.d/org.fwupd.hsi.Amd.SpiReplayProtection.json:19]: @superm1 is capabilities the right word? plus, is there any existing way to identify CPUs like amd64 ix64 or something?

Comment (superm1) [reply docs/hsi-tests.d/org.fwupd.hsi.Amd.SpiReplayProtection.json:19]: > @superm1 is capabilities the right word?

I guess a better word might be `matching-hardware`?

> plus, is there any existing way to identify CPUs like amd64 ix64 or something?

What do you mean?  Identify from where?

Comment (hughsie) [reply docs/hsi-tests.d/org.fwupd.hsi.Amd.SpiReplayProtection.json:19]: Ohh, i mean an industry standard token, e.g. CPU_AMD64 or something defined in the kernel maybe.

Comment (superm1) [reply docs/hsi-tests.d/org.fwupd.hsi.Amd.SpiReplayProtection.json:19]: read `/proc/cpuinfo` and look at `vendor_id`.

Comment (hughsie) [reply docs/hsi-tests.d/org.fwupd.hsi.Amd.SpiReplayProtection.json:19]: so how about `s/cpu-amd/CPUID:AuthenticAMD`?

Comment (superm1) [reply docs/hsi-tests.d/org.fwupd.hsi.Amd.SpiReplayProtection.json:19]: GUIDs is a great idea.

**Review by superm1 — APPROVED** [review]

**Pull request merged**